apps/aehttp/test/aehttp_hyperchains_SUITE.erl

```diff
@@ -818,6 +820,33 @@ epochs_with_fast_parent(Config) ->
     %% Here we should be able to observe signalling that epoch should be shorter
     ok.
 
+%%%=============================================================================
+%%% HC Endpoints
+%%%=============================================================================
+
+get_contract_pubkeys(Config) ->
+    [{Node, _, _} | _] = ?config(nodes, Config),
+    %% Verify that endpoint is available
+    {ok, IsChildChain} = rpc(Node, aeu_env, find_config,
+                             [[<<"http">>, <<"endpoints">>, <<"hyperchain">>], [user_config, schema_default]]),
+    ?assert(IsChildChain),
+    StakingContractPK = rpc(Node, aec_consensus_hc, get_contract_pubkey, [staking]),
+    ElectionContractPK = rpc(Node, aec_consensus_hc, get_contract_pubkey, [election]),
+    RewardsContractPK = rpc(Node, aec_consensus_hc, get_contract_pubkey, [rewards]),
+    %% note: the pins are for the last block in previous epoch
```

Comment:
I'm not entirely sure what this has to do with the code...

Reply:
I test what I get back from the call is identical to what was in the contract?

Reply:
But good Q about that comment. C&P leftovers I presume. Gone now, will be in my next functional commit.

Reply:
The comment is still there, not resolved.